app/package.json

```diff
@@ -222,6 +222,7 @@
     "react-syntax-highlighter": "^10.1.2",
     "react-toggle-switch": "^3.0.4",
     "react-transition-group": "^2.2.1",
+    "react-vis": "^1.11.7",
```

Comment:
While from a quick glance I definitely like `react-vis` more than `highcharts`, I wonder if we should use `highcharts` simply because we already have it bundled. 

https://bundlephobia.com/result?p=react-vis@1.11.7

![image](https://user-images.githubusercontent.com/43438/66804705-ca7da000-ef55-11e9-9994-6142425b3728.png)

Though I would like to move away from `highcharts` so maybe we should just plan to replace `highcharts` with `react-vis` for the chart component? Either way we should probably only use a single charting lib.

Reply:
Yeah, would be nice to have single charting library. I picked up `react-vis` because it was used on the homepage and had good baseline for styling already set up.